src/test/java/GetUserByIdCommandTest.java

```diff
@@ -0,0 +1,37 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+
+class GetUserByIdCommandTest {
+
+    @Test
+    void shouldExecuteIfUserIdWasFound() {
+        UsersBook usersBook = new UsersBook();
+        TestConsole console = new TestConsole(1);
+        User user = new User("name", "surname", 1);
+        usersBook.addUser(user);
+
+        GetUserByIdCommand getUserByIdCommand = new GetUserByIdCommand(usersBook, console);
+
+        boolean consoleMessageBeforeExecuted = console.messages.isEmpty();
+        Assertions.assertTrue(consoleMessageBeforeExecuted);
+
+        getUserByIdCommand.execute();
+
+        boolean consoleMessageAfterExecuted = console.messages.isEmpty();
+        Assertions.assertFalse(consoleMessageAfterExecuted);
```

Comment:
no need check that console is not empty. Actually you just need to check that console contains some message. If it is contains, then it is guarantee is not empty. Otherwise it is just test fail in both cases